.env.dev

```diff
@@ -4,4 +4,6 @@ AUTH_CERTS_URL=http://moonshot-dev.mitre.org:8090/auth/realms/backend_app/protoc
 AUTH_TOKEN_URL=http://moonshot-dev.mitre.org:8090/auth/realms/backend_app/protocol/openid-connect/token
 DATA_TRUST_SERVICE=http://localhost:3005
 BASE_URL=http://localhost:3000
-ADMIN_TOKEN=admin
\ No newline at end of file
+ADMIN_TOKEN=admin
+REQUIRE_AUTH=false
```

Comment:
Two quick things - can you make the default here `true` and add these environment variables to `docker-compose.yml` and `docker/aws-docker-compose.yml` files?